tools/osbuilder/rootfs-builder/ubuntu/Dockerfile.in

```diff
@@ -14,18 +14,21 @@ RUN apt-get update && \
     ca-certificates \
     curl \
     g++ \
-    $(gcc_arch="@ARCH@" && [ "$(uname -m)" != "$gcc_arch" ] && ( \
+    "$(gcc_arch="@ARCH@" && [ "$(uname -m)" != "$gcc_arch" ] && ( \
          libc_arch="$gcc_arch" && \
          [ "$gcc_arch" = aarch64 ] && libc_arch=arm64; \
          [ "$gcc_arch" = ppc64le ] && gcc_arch=powerpc64le && libc_arch=ppc64el; \
+         [ "$gcc_arch" = s390x ] && gcc_arch=s390x && libc_arch=s390x; \
          [ "$gcc_arch" = x86_64 ] && gcc_arch=x86-64 && libc_arch=amd64; \
-         echo "gcc-$gcc_arch-linux-gnu libc6-dev-$libc_arch-cross")) \
+         echo "gcc-${gcc_arch}-linux-gnu libc6-dev-${libc_arch}-cross"))" \
     git \
     make \
     makedev \
     multistrap \
     musl-tools \
-    protobuf-compiler
+    protobuf-compiler \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
This leads to the following error during `rootfs-*` build:

```
+ multistrap -a s390x -d /rootfs -f multistrap.conf                                                                                                                                                                
multistrap 2.2.10 using multistrap.conf                                                                                                                                                                            
multistrap 2.2.10 using multistrap.conf                                                                                                                                                                            
Using foreign architecture: s390x                                                                                                                                                                                  
multistrap building s390x multistrap on 'amd64'                                                                                                                                                                    
I: Setting /rootfs/lib64 -> /rootfs/lib symbolic link.                                                                                                                                                             
I: Downloading ubuntu-keyring                                                                                                                           
E: Can't find a source to download version '2020.02.11.4' of 'ubuntu-keyring:amd64'                                                                                                                                
multistrap: Unable to download keyring package: '/rootfs/'
```

Reply:
Yeah, these 2 change is recommended by dockerfile lint check. Without these, it will give a warning.  I remove these first as correctness first.